cypress.config.js

```diff
@@ -19,6 +19,7 @@ const storeData = async (data, filepath) => {
 }
 
 export default defineConfig({
+  projectId: 'fg82nr',
```

Comment:
I guess that's the mfe e2e tests project id mentioned within that massive discussion thread on housing channel?

Reply:
pretty much